mpa/cls/trainer.py

```diff
@@ -161,8 +165,8 @@ def train_worker(gpu, dataset, cfg, distributed, validate, timestamp, meta):
                 sub_loaders = [
                     build_dataloader(
                         sub_ds,
-                        sub_ds.samples_per_gpu if hasattr(sub_ds, 'samples_per_gpu') else cfg.data.samples_per_gpu,
-                        sub_ds.workers_per_gpu if hasattr(sub_ds, 'workers_per_gpu') else cfg.data.workers_per_gpu,
+                        sub_ds.samples_per_gpu,
```

Comment:
Will all tests pass even if the attr check disappears?

Reply:
For supervised/incremental learning, there was no problem. I'll try to catch edge cases for these lines.

Reply:
These lines are also updated, please review again. Thank you.